src/shared/navgen/nav.cpp

```diff
@@ -308,27 +319,22 @@ void NavmeshGenerator::LoadBrushTris( std::vector<float> &verts, std::vector<int
 			}
 		}
 	}
-}
-
-static void LoadPatchTris( std::vector<float> &verts, std::vector<int> &tris ) {
-// Disabling this code for now. It should be straightforward to port, but I haven't
-// bothered because I can't see any effect on the navmesh, even on parts with curved
-// areas using patches. So maybe we are better of dropping this and saving some triangles.
-	Q_UNUSED(verts);
-	Q_UNUSED(tris);
-#if 0
-	vec3_t mins, maxs;
-	int solidFlags = 0;
-	int temp = 0;
 
-	char surfaceparm[16];
-
-	strcpy( surfaceparm, "default" );
-	ApplySurfaceParm( surfaceparm, &solidFlags, NULL, NULL );
+	if ( !config_.generatePatchTris )
+	{
+		return;
+	}
+	LOG.Debug( "Generated %d brush tris", tris.size() / 3 );
+	LOG.Debug( "Generated %d brush verts", verts.size() / 3 );
```

Comment:
We might want to put this above (nit)

Reply:
The total number of tris is already logged elsewhere so it only provides non-duplicate information if patches are enabled